### PR TITLE
LibOpenMPT: bump

### PR DESCRIPTION
**Pull request by extrowerk**
Soname changed, thus libopenmpt4 introduced for backwards compatibility.

### CODE REVIEW

**Comment by korli** [comment]
@extrowerk  why soname change? libopenmptVersion stays zero. Is lib:libopenmpt_modplug used anywhere?

**Comment by extrowerk** [comment]
@korli:
>why soname change? libopenmptVersion stays zero. 
My mistake then, sorry.
>Is lib:libopenmpt_modplug used anywhere?
No it isn1t used anywhere, and it isn't provided by openmpt anymore, it became an own separate package, will do a recipe for that too in case we want to ditch libmodplug completely.

**Comment by extrowerk** [comment]
I think no change required then here, right @korli ?

**Comment by korli** [comment]
> I think no change required then here, right @korli ?

libopenmpt4 isn't needed.

**Review by korli — APPROVED** [review]

**Comment by Begasus** [comment]
libopenmpt has been bumped to 0.5.3 in the meantime :)

File: media-libs/libopenmpt/libopenmpt-0.5.2.recipe [hunk 1]
```diff
@@ -3,25 +3,22 @@ DESCRIPTION="libopenmpt is a cross-platform C++ and C module playback \
...
 REVISION="1"
 SOURCE_URI="https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-$portVersion+release.autotools.tar.gz"
-CHECKSUM_SHA256="260e92cc2f6af37113442bff2c75a3c36a09eba4078dc593203a0502f95d26bd"
+CHECKSUM_SHA256="9468e3e501859821a5a3da76b8cceca2728ed3cbc8a818b550fd3690246197b3"
 SOURCE_DIR="libopenmpt-$portVersion+release.autotools"
 
-ARCHITECTURES="?x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```
Comment (Begasus) [inline media-libs/libopenmpt/libopenmpt-0.5.2.recipe:15]: Maybe?
```diff
- SECONDARY_ARCHITECTURES="x86"
```

**Pull request merged**